composer.json

```diff
@@ -14,7 +14,7 @@
         "discard-changes": true
     },
     "require" : {
-        "php" : ">=5.6.0",
+        "php" : ">=7.2.0",
```

Comment:
Which feature requires PHP 7.2? The `??` operator seems to have been introduced in PHP 7.0?

Reply:
Because PHP 7.0 and 7.1 were also outdated and not supported anymore. Why use another old version and give up new features that might be used. https://www.php.net/supported-versions.php